packages/frontend/component/src/components/setting-components/workspace-detail-skeleton.tsx

```diff
@@ -8,19 +8,17 @@ export const WorkspaceDetailSkeleton = () => {
     <>
       <SettingHeader title={<Skeleton />} subtitle={<Skeleton />} />
 
-      {Array.from({ length: 3 })
```

Comment:
you can also move the `map` fn into the second argument of `Array#from` to make it shorter